nilearn/glm/contrasts.py

```diff
@@ -262,6 +265,37 @@ def p_value(self, baseline=0.0):
         self.p_value_ = p_values
         return p_values
 
+
+    def one_p_value(self, baseline=0.0):
```

Comment:
I don't understand why it is necessary to introduce a new function.

Reply:
Oh, I understand, this is one_minus_pvalue !
Please rename this !

Reply:
OK, I have renamed it. It is now called `one_minus_pvalue`.